test/fixtures/all-secrets.yaml

```diff
@@ -0,0 +1,16 @@
+sources:
```

Comment:
Here's the `secrets.yaml` that corresponds to the `config.yaml`. Source secrets, like API keys, are referenced by the `org_id`, and sink secrets are referenced by the `sink.id`.